dashboard/app/views/levels/_admin.html.haml

```diff
@@ -36,7 +36,7 @@
         %li (Cannot edit)
 
     %br/
-    This level is in #{@level.script_levels.count} scripts:
+    This level is in #{@level.script_levels.to_a.count} scripts:
```

Comment:
This is presumably unrelated (preventing `count` from causing a DB dip), so could (should) be a separate PR.